jsk_pcl_ros_utils/test/pointcloud_to_pcd.test

```diff
@@ -1,12 +1,11 @@
 <launch>
-  <node name="rosbag_play"
-        pkg="rosbag" type="play"
-        args="$(find jsk_pcl_ros_utils)/test_data/2015-11-04-19-37-29_baxter-kiva-object-in-hand-cloud/vision.compressed.bag --clock --loop">
-  </node>
+
   <include file="$(find jsk_pcl_ros_utils)/sample/sample_pointcloud_to_pcd.launch">
-      <arg name="INPUT" value="/kinect2_head/hd/points" />
+    <arg name="PREFIX_PATH" value="$(find jsk_pcl_ros_utils)/test_data/test_pcd_" />
```

Comment:
I think they should be located in `~/.ros` because if we install data to pkg_path, we need to run with `sudo` when we install it with apt.